src/dotnet-roslyn-tools/CreateReleaseTags/CreateReleaseTags.cs

```diff
@@ -47,43 +48,43 @@ public static async Task<int> CreateReleaseTagsAsync(RoslynToolsSettings setting
 
             logger.LogInformation("Tagging releases...");
 
-            foreach (var visualStudioRelease in visualStudioReleases)
+            foreach (var visualStudioRelease in visualStudioReleases.Reverse())
             {
-                var roslynTagName = TryGetRoslynTagName(visualStudioRelease);
+                var tagName = TryGetTagName(visualStudioRelease);
 
-                if (roslynTagName is not null)
+                if (tagName is not null)
                 {
-                    if (!existingTags.Any(t => t.FriendlyName == roslynTagName))
+                    if (!existingTags.Any(t => t.FriendlyName == tagName))
                     {
-                        logger.LogInformation($"Tag '{roslynTagName}' is missing.");
+                        logger.LogInformation($"Tag '{tagName}' is missing.");
 
-                        RoslynBuildInformation? roslynBuild = null;
+                        RoslynBuildInformation? build = null;
                         foreach (var connection in connections)
                         {
-                            roslynBuild = await TryGetRoslynBuildForReleaseAsync(visualStudioRelease, devdivConnection, connection);
+                            build = await TryGetBuildInfoForReleaseAsync(product, visualStudioRelease, devdivConnection, connection);
 
-                            if (roslynBuild is not null)
+                            if (build is not null)
                             {
                                 break;
                             }
                         }
 
-                        if (roslynBuild is not null)
+                        if (build is not null)
                         {
-                            logger.LogInformation($"Tagging {roslynBuild.CommitSha} as '{roslynTagName}'.");
+                            logger.LogInformation($"Tagging {build.CommitSha} as '{tagName}'.");
 
-                            string message = $"Build Branch: {roslynBuild.SourceBranch}\r\nInternal ID: {roslynBuild.BuildId}\r\nInternal VS ID: {visualStudioRelease.BuildId}";
+                            string message = $"Build Branch: {build.SourceBranch}\r\nInternal ID: {build.BuildId}\r\nInternal VS ID: {visualStudioRelease.BuildId}";
 
-                            roslynRepository.ApplyTag(roslynTagName, roslynBuild.CommitSha, new Signature("dotnet bot", "dotnet-bot@microsoft.com", when: visualStudioRelease.CreationTime), message);
+                            repository.ApplyTag(tagName, build.CommitSha, new Signature("dotnet bot", "dotnet-bot@microsoft.com", when: visualStudioRelease.CreationTime), message);
```

Comment:
Considering earlier question around whether TypeScript/F# could use this -- would they use something other than dotnet-bot?